src/Tizen.Network.WiFiDirect/Tizen.Network.WiFiDirect/WiFiDirectManager.cs

```diff
@@ -1251,6 +1251,38 @@ public static void CreateGroup()
             }
         }
 
+        /// <summary>
+        /// Creates a Wi-Fi Direct group with given SSID and sets up device as the group owner.
+        /// </summary>
+        /// <privilege>
+        /// http://tizen.org/privilege/wifidirect
+        /// </privilege>
+        /// <feature>
+        /// http://tizen.org/feature/network.wifidirect
+        /// </feature>
+        /// <remarks>
+        /// Wi-Fi Direct must be activated.
+        /// If this succeeds, ConnectionStatusChanged event will be invoked with GroupCreated.
```

Comment:
please consider making those references clickable:
```suggestion
        /// If this succeeds, <see cref="ConnectionStatusChanged"/> event will be invoked with <see cref="WiFiDirectConnectionState.GroupCreated"/>.
```